.github/workflows/test_and_deploy.yml

```diff
@@ -32,14 +33,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       # these libraries enable testing on Qt on linux
-      - uses: tlambert03/setup-qt-libs@v1
-
-      # strategy borrowed from vispy for installing opengl libs on windows
-      - name: Install Windows OpenGL
-        if: runner.os == 'Windows'
-        run: |
-          git clone --depth 1 https://github.com/pyvista/gl-ci-helpers.git
-          powershell gl-ci-helpers/appveyor/install_opengl.ps1
```

Comment:
Do we need/want this Windows thing?

Reply:
I was just updating the workflow to match the [current version](https://github.com/napari/cookiecutter-napari-plugin/blob/main/%7B%7Bcookiecutter.plugin_name%7D%7D/.github/workflows/test_and_deploy.yml) in current napari-plugin. But I think you're right that it's not necessary - I'll remove it

Reply:
ah sorry I have already removed it! Do you want me to add it back?

Reply:
If it's not needed, happy for it to go